frontend/src/lib/lemon-ui/LemonTable/TableCellSparkline.tsx

```diff
@@ -1,49 +1,36 @@
-import { useEffect, useMemo, useRef, useState } from 'react'
-import { useValues } from 'kea'
+import { useEffect, useRef, useState } from 'react'
 import { Chart, ChartItem, TooltipModel } from 'lib/Chart'
-import { range } from 'lib/utils'
-import { dayjs, dayjsUtcToTimezone } from 'lib/dayjs'
-import { teamLogic } from '../../teamLogic'
-import { IngestionWarningSummary } from './ingestionWarningsLogic'
 import { Popover } from 'lib/lemon-ui/Popover/Popover'
 import { offset } from '@floating-ui/react'
 
-import './WarningEventsGraph.scss'
+import './TableCellSparkline.scss'
 
-export function WarningEventsGraph({ summary }: { summary: IngestionWarningSummary }): JSX.Element {
+export interface SparklineDataset {
+    dates?: string[]
+    data: number[]
+}
```

Comment:
I think these should be the component's props. I don't really see why we need to pass them through a `dataset` property. Plus, there's a re-rendering issue like this, partly my fault... see next comment.